src/Arcus.Scripting.Storage.Table/Arcus.Scripting.Storage.Table.psm1

```diff
@@ -51,4 +51,52 @@ function Create-AzStorageTable {
     }
 }
 
-Export-ModuleMember -Function Create-AzStorageTable
\ No newline at end of file
+Export-ModuleMember -Function Create-AzStorageTable
+
+
+
+<#
+ .Synopsis
+  Sets the entities in a specified table storage.
+  
+ .Description
+  Sets (Delete & Add) the entities in a specified table storage to the values specified in a (JSON) configuration file.
+
+  .Parameter ResourceGroupName
+  The resource group where the Azure Storage Account is located.
+
+ .Parameter StorageAccountName
+  The name of the Azure Storage Account which holds the Azure table storage.
+
+ .Parameter TableName
+  The name of the table on the Azure Storage Account.
+
+ .Parameter ConfigurationFile
+  The Configuration File (JSON) specifying the entities that have to be set in the table storage. The JSON file has a single property "entities" which is an array of objects.
+
+ .Parameter RetryIntervalSeconds
+  The optional amount of seconds to wait each retry-run when a failure occures during the re-creating process.
+
+ .Parameter MaxRetryCount
+  The optional maximum amount of retry-runs should happen when a failure occurs during the re-creating process.
```

Comment:
These parameters are not used in the script, and are also not documented, they can be removed?

Reply:
Good one, I removed them from the script but forgot to remove them from the psm1 file. Fixed in [`cdb36fd` (#222)](https://github.com/arcus-azure/arcus.scripting/pull/222/commits/cdb36fd093dd526f86cdbf235840db847fe81c63)